<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>511</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>421</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>57</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Minutes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Seconds</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>48</width>
      <height>54</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="maximum">
      <number>240</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_4">
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Segment start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Segment end</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>48</width>
      <height>54</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>240</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>57</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Minutes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Seconds</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality settings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>57</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>crf</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>codec</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>90</y>
      <width>51</width>
      <height>54</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSpinBox" name="spinBox_5">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>64</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>208</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Input file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_configuration"/>
    <addaction name="actionRead_configuration"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInformation"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_configuration">
   <property name="text">
    <string>Save configuration</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRead_configuration">
   <property name="text">
    <string>Read configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
